File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/PhotosModel/OpeningHours.java

package restaurant.sa.com.sarestaurant.appview.restaurant.model.PhotosModel;

import java.util.List;
import com.google.gson.annotations.SerializedName;
public class OpeningHours {

    @SerializedName("open_now")
    private Boolean mOpenNow;
    @SerializedName("periods")
    private List<Period> mPeriods;
    @SerializedName("weekday_text")
    private List<String> mWeekdayText;

    public Boolean getOpenNow() {
        return mOpenNow;
    }

    public void setOpenNow(Boolean openNow) {
        mOpenNow = openNow;
    }

    public List<Period> getPeriods() {
        return mPeriods;
    }

    public void setPeriods(List<Period> periods) {
        mPeriods = periods;
    }

    public List<String> getWeekdayText() {
        return mWeekdayText;
    }

    public void setWeekdayText(List<String> weekdayText) {
        mWeekdayText = weekdayText;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/PhotosModel/OpeningHours.kt
package restaurant.sa.com.sarestaurant.appview.restaurant.model.PhotosModel

import com.google.gson.annotations.SerializedName

class OpeningHours {

    @SerializedName("open_now")
    var openNow: Boolean? = null
    @SerializedName("periods")
    var periods: List<Period>? = null
    @SerializedName("weekday_text")
    var weekdayText: List<String>? = null

}
